app/models/survey_execution_state.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: survey_execution_states
+#
+#  id                                   :integer          not null, primary key
+#  question_id                          :integer
+#  respondent_id                        :integer
+#  created_at                           :datetime         not null
+#  updated_at                           :datetime         not null
+#
+
+class SurveyExecutionState < ActiveRecord::Base
+  belongs_to :respondent
+  belongs_to :question
```

Comment:
Given that the model name has the word survey in it, might make sense to expose the associated survey. (More of a nice-to-have)